Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -411,16 +439,31 @@ def refresh_session(host_id: str) -> Dict:
     return response
 
 
-def run_batch_read_cmd(host_ids: list, command_type: str, full_command: str) -> Dict:
+def batch_refresh_session(batch_id: str) -> None:
+    """
+        Batch refresh a RTR session on multiple hosts.
+        :param batch_id:  Batch ID to execute the command on.
+    """
+    demisto.debug('Starting session refresh')
+    endpoint_url = '/real-time-response/combined/batch-refresh-session/v1'
+
+    body = json.dumps({
+        'batch_id': batch_id
+    })
+    http_request('POST', endpoint_url, data=body)
```

Comment:
Maybe we should check the response and log it if it is an error?

Reply:
if there is an error it should be caught by the error handling in `http_request`
i will log the response anyway